web/src/components/SharingPopup.tsx

```diff
@@ -208,14 +209,14 @@ const SharingPopup = ({ shareUrl, title, flow, direction, isWithinPortal }: Shar
     <SharingPopupContainer>
       {shareOptionsVisible && (
         <>
-          {isWithinPortal && (
+          {portalNeeded && (
```

Comment:
Actually when do we need the portal? Why do we need the backdrop, when the poifeedback is fullscreen for the bottom action sheet?

Reply:
The backdrop is for the sharing popup, not the feedback. I dont know what exactly the reason is but the bottom action sheet is doing some weird stuff that breaks some other styling...

Reply:
Ah. You're right. Well then you're good to go :)